From 62de8a4fa0b278e7c1551be87cc440a0184eef2f Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 19 Feb 2019 15:28:39 +0800
Subject: [PATCH] arch/arm/boot/dts: change node name from pciex@xxx to
 pcie@xxx

This patch fix compiling warning as below:

arch/arm/boot/dts/axm5516-amarillo.dtb: Warning (pci_bridge): /soc/pciex@0x3000000000: node name is not "pci" or "pcie"
arch/arm/boot/dts/axm5516-amarillo.dtb: Warning (pci_bridge): /soc/pciex@0x3080000000: node name is not "pci" or "pcie"
arch/arm/boot/dts/axm5516-amarillo.dtb: Warning (pci_device_bus_num): Failed prerequisite 'pci_bridge'

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm/boot/dts/axm55xx.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/axm55xx.dtsi b/arch/arm/boot/dts/axm55xx.dtsi
index 03be6eb..0cb5636 100644
--- a/arch/arm/boot/dts/axm55xx.dtsi
+++ b/arch/arm/boot/dts/axm55xx.dtsi
@@ -176,7 +176,7 @@
 			reg = <0x20 0x10094000 0 0x1000>;
 		};
 
-		pcie0: pciex@0x3000000000 {
+		pcie0: pcie@0x3000000000 {
 			compatible = "axxia,plb-pciex";
 			device_type = "pci";
 			#interrupt-cells = <1>;
@@ -217,7 +217,7 @@
 			status = "disabled";
 		};
 
-		pcie1: pciex@0x3080000000 {
+		pcie1: pcie@0x3080000000 {
 			compatible = "axxia,plb-pciex";
 			device_type = "pci";
 			#interrupt-cells = <1>;
-- 
1.7.9.5

